users/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+
+class User(models.Model):
+    identification      =   models.CharField(max_length=45)
+    password            =   models.CharField(max_length=500)
+    name                =   models.CharField(max_length=45)
+    email               =   models.CharField(max_length=100)
+    phone_number        =   models.CharField(max_length=45)
+    birthdate           =   models.CharField(max_length=45)
+    gender              =   models.CharField(max_length=10)
+    product             =   models.ManyToManyField('products.Product',through = 'UserLike')
+    created_at          =   models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table    = 'users'
+
+class Address(models.Model):
+    address             =   models.CharField(max_length=500)
+    user                =   models.ForeignKey('User',on_delete = models.CASCADE)
+    is_default          =   models.SmallIntegerField()
```

Comment:
```suggestion
    is_default          =   models.BooleanField(default=False)
```
BooleanField가 있습니다! 
https://docs.djangoproject.com/en/3.2/ref/models/fields/#booleanfield

Reply:
BolleanField로 수정했습니다~